hash/risc0/bench/bench.cfg

```diff
@@ -0,0 +1,12 @@
+name:   "Hashes benchmarking using risc0 prover"
+author: 
+timeout: 100
+params:
+  [ PROVER:     [ risc0 ]
```

Comment:
so the point of this file, is to specify (for the automatic runner), what parameters the benchmark (run.sh) understands. So these must match those, and list values we want to run.

since there is no `ZKBENCH_PROVER` in `run.sh`, you don't want that here. However, there is `ZKBENCH_HASH_TYPE`, so we need `HASH_TYPE` here, and list `[ "sha256", "keccak", "blake2b", ...]`

same for the others. For the tree-based ones, we will need a second `.cfg` file, say `bench_tree.cfg` because they have different parameters (tree depth instead of bytes)

The tags are intended to filter the benchmark numbers, as there will be too many of them to display or plot in a useful way. So the idea is that instead of listing all the hash types, you just write `$HASH_TYPE`, so the SHA256 result will have tag SHA256 only, the keccak one will have tag KECCAK, etc.

At some point there will be a user interface where you can use these tags to select a subset of benchmark results and display them.